end-to-end-tests/specs/results.spec.js

```diff
@@ -135,6 +135,29 @@ describe('Results Page', function() {
 
     });
 
+    describe("Mutations Tab", () => {
+
+        describe('3D structure visualizer', () => {
+            before(() => {
+                var url = `${CBIOPORTAL_URL}/index.do?tab_index=tab_visualize&cancer_study_list=ov_tcga_pub&cancer_study_id=ov_tcga_pub&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=ov_tcga_pub_mutations&Z_SCORE_THRESHOLD=2.0&case_set_id=ov_tcga_pub_3way_complete&case_ids=&gene_list=BRCA1+BRCA2&gene_set_choice=user-defined-list&Action=Submit`;
+                browser.url(url);
+                browser.click("[href='#mutation_details']");
+                browser.waitForEnabled('[data-test=view3DStructure]', 10000);
+            });
+
+            it('populates PDB info properly', () => {
+                browser.click('[data-test=view3DStructure]');
+                browser.waitUntil(() => (browser.getText('[data-test=pdbChainInfoText]') !== "LOADING"), 10000);
```

Comment:
@alisman @inodb not sure if this is the best way to wait until the actual text loads

Reply:
@onursumer could use `waitForExist` but this works

Reply:
@inodb The problem is that `pdbChainInfoText` always exists, but its text content changes from `LOADING` to a real value. And the actual content is generated by another generic component called `TextExpander`. 

We could potentially render separate `div`s or `span`s for different states to be able to use `waitForExist` though.